.gitignore

```diff
@@ -162,7 +162,9 @@ cython_debug/
 .DS_Store
 
 #static files dirs
-staticfiles/
+/static
```

Comment:
`staticfiles/` is still confiugured as the STATIC_ROOT, so we still need it here

`STATIC_ROOT = BASE_DIR / "staticfiles"`